eclipse/org.mybatis.generator.eclipse.ui/src/org/mybatis/generator/eclipse/ui/content/ConfigVerifyer.java

```diff
@@ -97,6 +97,9 @@ public boolean isConfigurationFile() {
     private boolean isConfigFile(InputStream inputStream) {
         try {
             SAXParserFactory factory = SAXParserFactory.newInstance();
+            factory.setFeature("http://apache.org/xml/features/disallow-doctype-decl", true);
```

Comment:
This line breaks the functionality of the plugin. Please remove it.

Reply:
Removed this line.